codex-rs/core/src/codex.rs

```diff
@@ -1516,6 +1520,14 @@ async fn submission_loop(
                 };
                 sess.send_event(event).await;
             }
+            Op::RunUserShellCommand { command } => {
+                spawn_user_shell_command_task(
+                    sess.clone(),
+                    Arc::clone(&turn_context),
```

Comment:
This appears to be an unnecessary `clone()`.

```suggestion
                    &turn_context,
```